packages/parser-css/src/parser.ts

```diff
@@ -0,0 +1,68 @@
+import * as postcss from 'postcss';
+
+import { IAsyncHTMLElement, ElementFound, FetchEnd, Parser } from 'hint/dist/src/lib/types';
+import { StyleParse } from './types';
+import { Engine } from 'hint/dist/src/lib/engine';
+
+const styleContentRegex: RegExp = /^<style[^>]*>([\s\S]*)<\/style>$/;
+
+export default class CSSParser extends Parser {
+    public constructor(engine: Engine) {
+        super(engine, 'css');
+
+        engine.on('fetch::end::css', this.parseCSS.bind(this));
+        engine.on('element::style', this.parseStyleTag.bind(this));
+    }
+
+    private async emitCSS(code: string, resource: string) {
+
+        try {
+            const ast = postcss.parse(code, { from: resource });
+
+            const styleData: StyleParse = {
+                ast,
+                code,
+                resource
+            };
+
+            await this.engine.emitAsync(`parse::${this.name}::end`, styleData);
+
+        } catch (err) {
+            console.log(`Error parsing code: ${code} - ${err}`);
```

Comment:
We should be using `log.error` from utils or `debug`

Reply:
Ok. FYI parser-javascript does the same. Should we update that as well (possibly as a separate PR) or just leave it for now and fix this one?

Reply:
Separate PR I think should be fine. Not sure if we should use `debug` or `log.error` though. I think I'm more inclined towards `debug` but don't have a really strong opinion about it. @webhintio/core ?